### PR TITLE
refactor: post and comments permlink postfix #150

### PR DESCRIPTION
**Pull request by beautyfree**

### CODE REVIEW

File: src/components/modules/PostForm/PostForm.js [hunk 1]
```diff
@@ -513,7 +513,27 @@ export default class PostForm extends React.Component {
...
         Router.pushRoute('post', post.contentId);
       } else {
-        const result = await createPost(data);
+        let result;
+        try {
+          result = await createPost(data);
+        } catch (err) {
+          const message = normalizeCyberwayErrorMessage(err);
+
+          if (message.includes('This message already exists')) {
```
Comment (b1acksun) [inline src/components/modules/PostForm/PostForm.js]: :+1:

File: src/components/modules/PostForm/PostForm.js [hunk 2]
```diff
@@ -513,7 +513,27 @@ export default class PostForm extends React.Component {
...
+            const permlink = slug(`${data.title}-${new Date().getTime()}`, { lower: true });
+            try {
+              await this.handleSubmit({
+                ...data,
+                permlink,
+              });
+            } catch (err) {
+              throw err;
```
Comment (Bacher) [inline src/components/modules/PostForm/PostForm.js]: try/catch лишний

File: src/store/actions/complex/content.js [hunk 3]
```diff
@@ -67,7 +67,7 @@ export const updatePost = ({ contentId, title, body, tags, jsonmetadata, resourc
 export const createComment = ({ contentId, body, jsonmetadata, resources, curationPercent }) => {
   const data = {
     message_id: {
-      permlink: `re-${contentId.permlink}`,
+      permlink: `re-${contentId.permlink}-${new Date().getTime()}`,
```
Comment (Bacher) [inline src/store/actions/complex/content.js]: Date.now()

File: src/components/modules/PostForm/PostForm.js [hunk 4]
```diff
@@ -513,7 +513,27 @@ export default class PostForm extends React.Component {
...
       } else {
-        const result = await createPost(data);
+        let result;
+        try {
+          result = await createPost(data);
+        } catch (err) {
+          const message = normalizeCyberwayErrorMessage(err);
+
+          if (message.includes('This message already exists')) {
+            const permlink = slug(`${data.title}-${new Date().getTime()}`, { lower: true });
```
Comment (Bacher) [inline src/components/modules/PostForm/PostForm.js]: Date.now()

**Pull request merged**